src/scss/components/image-fundation.scss

```diff
@@ -0,0 +1,50 @@
+.image-fundation {
+  background-image: url('../static/assets/Forma-image-fundation.png');
+  background-position-x: 100%;
+  background-position-y: 100%;
+  background-repeat: no-repeat;
+  margin-bottom: 54.2px;
```

Comment:
Please add a `margin-top` here as well, so that in mobile there is a space between the previous component and this one.

Reply:
we remained in the project that the one who placed the distance was always the component above.
Similarly,  is it necessary to place the margin top?

Reply:
Ok. In that case, just set the margin-bottom